CMakeLists.txt

```diff
@@ -113,7 +113,14 @@ set(ua_architecture_headers ${ua_architecture_headers}
     ${PROJECT_SOURCE_DIR}/include/open62541/architecture_functions.h
 )
 
+if(UA_ENABLE_WEBSOCKET_SERVER)
+    add_definitions(-DUA_ENABLE_WEBSOCKET_SERVER)
```

Comment:
This should be put into `include/open62541/config.h.in`, because then it will always be defined automatically and you do not have to add it to every gcc call.

I'll update this PR and then merge it.